docs/2.7.0/docs/canton/usermanual/security.rst

```diff
@@ -476,15 +494,62 @@ You can optionally pass a wrapper key id to change to or let Canton generate a n
 KMS configuration. If you wish to change the key specification (e.g. enable multi region) you are required
 to update the configuration before rotating the wrapper key.
 
+Canton Configuration for External Key Storage and Usage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    You cannot mix an external private key storage configuration
+    with an encrypted private key storage configuration.
+
+In the example below we configure a Canton participant node (called ``participant1``) to generate and
+store private keys in an external KMS. Besides the previously presented :ref:`AWS KMS configuration <kms_config>`
+you only need to specify the correct crypto provider ``kms``:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/encrypted-store-enabled-tagged.conf
+   :language: none
+   :start-after: user-manual-entry-begin: PrivateKeyStoreConfig
+   :end-before: user-manual-entry-end: PrivateKeyStoreConfig
+
+AWS KMS only supports the following cryptographic schemes:
```

Comment:
refer to the section with the table of supported schemes, we don't need to repeat it here